Assets/LeapMotion/Core/Resources/HandData.cginc

```diff
@@ -0,0 +1,311 @@
+	dist = min(dist, Leap_Dist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[1])));
+	dist = min(dist, Leap_Dist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[2])));
+	dist = min(dist, Leap_Dist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[3])));
+	dist = min(dist, Leap_Dist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[4])));
+	return dist;
+}
+
+float Leap_SqrDistToFingertips(float3 pos) {
+	float dist =     Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[0]));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[1])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[2])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[3])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[4])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[0])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[1])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[2])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[3])));
+	dist = min(dist, Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[4])));
+	return dist;
+}
+
+float Leap_SqrDistToFingertips_WithScale(float3 pos, float3 scale) {
+	float dist =     Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[0]) * scale);
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[1]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[2]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[3]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[4]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[0]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[1]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[2]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[3]) * scale));
+	dist = min(dist, Leap_SqrDist(pos * scale, mul(Leap_HandData_Preprocess_Matrix, _Leap_RH_Fingertips[4]) * scale));
+	return dist;
+}
+
+float Leap_SqrDistToHand(float3 pos) {
+  // Fingertips
+	//float dist =     Leap_SqrDist(pos, mul(Leap_HandData_Preprocess_Matrix, _Leap_LH_Fingertips[0]));
+  //for (int i = 1; i < 5; i++) {
```

Comment:
Lots of commented out code here, still needed?

Reply:
I don't remember :D will see if it's useful for anything, might have been an early version that only checked fingertip positions